### PR TITLE
[WIP] Landing page prototype

### PR DESCRIPTION
**Pull request by arnavb**
This is only a design prototype, not final.

#2.

Issues at the moment:
- ~~Bulma fullheight hero + navbar is slightly taller than the viewport. Needs to be verified.~~ <-- Fixed in latest master.
- [ ] Landing page info is still missing.

### CODE REVIEW

**Comment by sagarreddypatil** [comment]
Why is build failing?

File: src/pages/home.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
 import React from "react";
 
 export default function Home() {
-  return <h2>Home</h2>;
+  return (
+    <>
+      <section className="hero is-link is-fullheight-with-navbar">
+        <div className="hero-body">
+          <div className="container">
+            <p className="title">InvisionAR</p>
```
Comment (sagarreddypatil) [inline src/pages/home.js:9]: V should be capitalized like "InVisionAR"

File: package.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "bulma": "^0.9.0",
+    "node-sass": "^4.14.1",
```
Comment (sagarreddypatil) [inline package.json]: Move to dev dependency

File: src/pages/home.js (hunk 1 above)
Comment (sagarreddypatil) [inline src/pages/home.js:9]: V should be capitalized

**Review by sagarreddypatil — CHANGES_REQUESTED** [review]
Need to add the actual pages and fix failing ci build

**Comment by sagarreddypatil** [comment]
bulma has a fullheight hero. docs here: https://bulma.io/documentation/layout/hero/

**Comment by arnavb** [comment]
@sagarreddypatil I fixed it. Don't forget we have a navbar also.

**Comment by arnavb** [comment]
Superseded by #9, #10, and #11

**Pull request closed without merging**